studies/india_districts/etl.py

```diff
@@ -168,6 +172,24 @@
     'Num cases'
 ]
 
+current_version = 4
+
+def download_data(data_path: Path):
+    base_url = 'https://api.covid19india.org/csv/latest/'
+    today = datetime.today().strftime('%Y-%m-%d')
+
+    for i in range(current_version):
+        api_file = 'raw_data' + str(i + 1) + '.csv'
+        filename = today + '_' + api_file
+        if not os.path.exists(data_path/filename):
```

Comment:
`Path` objects also have `.exists()` and `.open()` methods - prefer consistency with `glob` below